### PR TITLE
Uno call

### PR DESCRIPTION
**Pull request by EricESeitz**

### CODE REVIEW

File: cards.js [hunk 1]
```diff
@@ -64,13 +64,27 @@ function deck(divId, hidden) {
     console.log(
       players[gameTurn].playerID + " Drew a " + randColor + " " + randValue
     ); //testing
+
+    //If drawing a card, player CANNOT have Uno
+    players[gameTurn].unoCall = 0;
```
Comment (CompSciLauren) [inline cards.js]: Can we use a boolean here instead?

**Review by TravisHull — APPROVED** [review]

**Review by CompSciLauren — CHANGES_REQUESTED** [review]
Only have 1 comment to address. Code looks good!

File: players.js [hunk 2]
```diff
@@ -4,18 +4,25 @@
...
  * @param {*} bot
+ * @param {*} unoCall
  */
-function player(deck, id, index, bot) {
+function player(deck, id, index, bot, unoCall) {
   this.isBot = bot;
   this.playerDeck = deck;
   this.playerID = id;
   this.playerIndex = index;
+  this.playerUnoCall = unoCall;
```
Comment (Earving8) [inline players.js:14]: You can just set this equal to 0 that way you don't have to pass in a parameter since Unocal just defaults to 0 on creation

**Review by Earving8 — CHANGES_REQUESTED** [review]

**Review by EricESeitz — COMMENTED** [review]
Changed unoCall to boolean

**Review by Earving8 — APPROVED** [review]

**Review by CompSciLauren — APPROVED** [review]

**Pull request merged**